resources/assets/components/ReviewBlock/test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import ReviewBlock from './index';
+
+test('it renders accepted, rejected, and delete buttons', () => {
+    const post = {
+      status: 'pending',
+    };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  expect(toJson(component)).toMatchSnapshot();
+});
+
+test('it renders the review block with tags', () => {
+    const post = {
+      status: 'accepted',
+    };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  expect(toJson(component)).toMatchSnapshot();
+});
+
+test('it renders an active Accept button when clicked', () => {
+  const post = {
+    status: 'pending',
+  };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+    // Click the "Accept" button.
+    component.find('.accepted').first().simulate('click');
+
+    // It should now show the active "Accept" button.
+    expect(component.find('.accepted').first().hasClass('button -accepted accepted'));
+});
+
+test('it renders an active Reject button when clicked', () => {
+  const post = {
+    status: 'pending',
+  };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  // Click the "Reject" button.
+  component.find('.rejected').first().simulate('click');
+
+  // It should now show the active "Reject" button.
+  expect(component.find('.rejected').first().hasClass('button -rejected rejected'));
+});
+
+test('it renders an active tag button when clicked', () => {
+  const post = {
+    status: 'pending',
+  };
+
+  const component = shallow(
+    <ReviewBlock post={post} />
+  );
+
+  // Click a tag button.
+  component.find('.tag').first().simulate('click');
+
+  // It should now show the active "Reject" button.
+  expect(component.find('.tag').first().hasClass('is-active'));
+});
+
```

Comment:
@DFurnes is it possible to do a test that checks if the `confirm` pop up happens when you click on the `Delete` button?

Reply:
Hmm, I think you can probably create a mock of that method on the `window`, kinda like we did with the parent component's `updateFilters` callback in `MultiValueFilter`:


```js
const confirmStub = sinon.stub('confirm', window);
confirmStub.returns(true);

// @TODO: simulate the click on the thing that asks for confirmation

expect(confirmStub.calledOnce);
```

Reply:
It looks like that might also be something to test on [`WithReviewing`](https://github.com/DoSomething/rogue/blob/31cf6bf3a8d882527dde1135d757a464756ffc0f/resources/assets/components/WithReviewing/index.js) instead (since that's where the actual `window.confirm` call is made), and just assert that the callback to that method is used here.

Reply:
@DFurnes ok that makes sense! Should I add to the list in [this story](https://www.pivotaltracker.com/n/projects/2019429/stories/150341644) to tackle on a separate PR?

Reply:
I think that makes sense! 👍